libs/agno/agno/app/fastapi/async_router.py

```diff
@@ -246,20 +249,44 @@ def team_process_file(
         return base64_images, base64_audios, base64_videos, document_files
 
     @router.post("/runs")
-    async def run_agent_or_team(
+    async def run_agent_or_team_or_workflow(
         message: str = Form(...),
         stream: bool = Form(False),
         monitor: bool = Form(False),
         session_id: Optional[str] = Form(None),
         user_id: Optional[str] = Form(None),
         files: Optional[List[UploadFile]] = File(None),
+        agent_id: Optional[str] = Query(None),
+        team_id: Optional[str] = Query(None),
+        workflow_id: Optional[str] = Query(None),
     ):
```

Comment:
While testing I noticed calling `http://localhost:8001/runs` (so, without the needed `?agent_id=...`) returns `{"detail": "'NoneType' object has no attribute 'deep_copy'"}` instead of communicating a required param is missing. Maybe an easy fix adjusting this?